### PR TITLE
Gear 522 basis vector inversion

### PR DESCRIPTION
**Pull request by joshicola**
The React version of the OHIF viewer inverted the y and the z axes.  This pull requests addresses
1. the axes inversions
2. proper incrementing of the bitmask labels

### CODE REVIEW

File: utils.py [hunk 1]
```diff
@@ -33,51 +33,53 @@ def poly2mask(vertex_row_coords, vertex_col_coords, shape):
...
+
     """Convert x,y point data into 3D masks.
 
     Args:
-        data (3D np array) the result
-        roi_image_path (string) provides orientation and slice info
-        roi_points (dict) the part of the roi dictionary that holds the point data
+        data (3D np array): The result
+        roi_image_path (string): Provides orientation and slice info
+        roi_points (dict): The part of the roi dictionary that holds the point data
```
Comment (AndysWorth) [inline utils.py:50]: Not actually an argument to get_points()?

File: utils.py [hunk 2]
```diff
@@ -33,51 +33,53 @@ def poly2mask(vertex_row_coords, vertex_col_coords, shape):
...
+        reactOHIF (bool, optional): Use React or Legacy Viewer. Defaults to True.
```
Comment (AndysWorth) [inline utils.py:51]: How about, "Defaults to React"?

File: utils.py [hunk 3]
```diff
@@ -161,16 +158,18 @@ def label2data(label, shape, info):
...
 
     # for OHIF REACT viewer:
-    elif ("ohifViewer" in info and
-          "measurements" in info["ohifViewer"] and
-          "FreehandRoi" in info["ohifViewer"]["measurements"]):
+    elif (
+        "ohifViewer" in info
+        and "measurements" in info["ohifViewer"]
+        and "FreehandRoi" in info["ohifViewer"]["measurements"]
+    ):
```
Comment (AndysWorth) [inline utils.py:169]: How about an "else" here that logs an error, maybe "unrecognized viewer" or something?  Otherwise it will fail silently.

**Comment by joshicola** [comment]
Andy, 
'black' PR-good idea.  It is hard to remember when it is automated in my editor.

File: utils.py (hunk 3 above)
Comment (joshicola) [reply utils.py:169]: Indeed,
If neither is present, we should indicate here (or earlier) that there is no data we can parse and exit.

NOTE: A WARNING is given earlier in `utils.gather_ROI_info()` when this is the case.  I'd like to warn the user only once on this matter.

File: utils.py (hunk 1 above)
Comment (joshicola) [reply utils.py:50]: roi_image_path -> image_path

File: utils.py (hunk 2 above)
Comment (joshicola) [reply utils.py:51]: Giving more context

**Comment by joshicola** [comment]
`nibabel` has a `numpy` requirement. Thus it is installed with `nibabel`.

**Pull request merged**